cognite/client/data_classes/transformations/common.py

```diff
@@ -276,10 +278,10 @@ def _load(cls, resource: dict[str, Any]) -> Instances:
 class OidcCredentials:
     def __init__(
         self,
-        client_id: str | None = None,
-        client_secret: str | None = None,
-        scopes: str | None = None,
-        token_uri: str | None = None,
+        client_id: str,
+        client_secret: str,
+        scopes: str,
+        token_uri: str,
```

Comment:
Breaking change, or it has earlier been caught by the API?

Reply:
Yup, it is breaking, but with any one of these missing (as in None), the transformation using them (the credentials) can't run. Thus, I think it should be a fair change, but open for a debate 😅

Reply:
Then, I also think it is ok. As this would fail later in the old version, you could not have passing code with passing None in to any of them.